lib/ansible/modules/monitoring/zabbix/zabbix_host_triggers.py

```diff
@@ -0,0 +1,272 @@
+                    clock: '1556182854'
+                    eventid: '630'
+                    value: '1'
+                  lastchange: '1556182854'
+                  priority: '4'
+                  state: '0'
+                  status: '0'
+                  templateid: '13615'
+                  triggerid: '14040'
+                  type: '0'
+                  url: ''
+                  value: '1'
+'''
+
+DOCUMENTATION = '''
+---
+module: zabbix_host_triggers
+short_description: Get all triggers about a Zabbix host
+description:
+   - This module allows you to see if a Zabbix host have no active alert to make actions on it.
+     For this case use module Ansible 'fail' to exclude host in trouble.
+   - Length of "triggers_ok" allow if template's triggers exist for Zabbix Host
+version_added: "2.9"
+author:
+    - "Stéphane Travassac (@stravassac)"
+requirements:
+    - "python >= 2.7"
+    - zabbix-api
+options:
+    server_url:
+        description:
+            - Url of Zabbix.
+        required: true
+    login_user:
+        description:
+            - Login of Zabbix.
+        required: true
+    login_password:
+        description:
+            - Password of Zabbix.
+        required: true
+    host_identifier:
+        description:
+            - Identifier of Zabbix Host
+        default: hostname
+        required: true
+    host_id_type:
+        description:
+            - Type of host_identifier
+        choices:
+            - hostname
+            - visible_name
+            - hostid
+        required: false
+        default: hostname
+    trigger_severity:
+        description:
+            - Zabbix severity for search filter
+        default: 3
+        required: false
+        type: int
+    timeout:
+        description:
+            - timeout before fail
+        default: 10
+        required: false
+        type: int
+    validate_certs:
+        description:
+                - SSL validate certificate
+        default: true
+        required: true
+        type: bool
+'''
+
+EXAMPLES = '''
+- name: exclude machine if alert active on it
+  zabbix_host_triggers_problem:
+      server_url: "{{ zabbix_url }}"
+      login_user: "{{ lookup('env','ZABBIX_USER') }}"
+      login_password: "{{ lookup('env','ZABBIX_PASSWORD') }}"
+      host_identifier: "{{inventory_hostname}}"
+      host_id_type: "hostname"
+      timeout: 120
+  register: zbx_host
+  delegate_to: localhost
+- fail:
+    msg: "machine alert in zabbix"
+  when: zbx_host['triggers_problem']|length > 0
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+
+try:
+    from zabbix_api import ZabbixAPI, ZabbixAPISubClass
+    # Extend the ZabbixAPI
+    # Since the zabbix-api python module too old (version 1.0, no higher version so far),
+    # it does not support the 'hostinterface' api calls,
+    # so we have to inherit the ZabbixAPI class to add 'hostinterface' support.
+
+    class ZabbixAPIExtends(ZabbixAPI):
```

Comment:
Extending of ZabbixAPI class was used with very old zabbix-api module versions. I am trying to get rid of this in #53334 as it is now working fine with just a base class. Please remove this as well. Your code will work fine without it :)